.chronus/changes/ModelExtendsIssue-2024-9-16-14-34-22.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: fix
+packages:
+  - "@typespec/openapi3"
+---
+
+template instance should not inline if it is response
```

Comment:
parameter also has the same behavior no?

Reply:
no